src/ResourceManager/Compute/Commands.Compute/Strategies/Network/SubnetStrategy.cs

```diff
@@ -23,32 +23,16 @@ static class SubnetStrategy
         public static NestedResourceStrategy<Subnet, VirtualNetwork> Strategy { get; }
             = NestedResourceStrategy.Create<Subnet, VirtualNetwork>(
                 provider: "subnets",
-                get: (vn, name) => vn.Subnets?.FirstOrDefault(s => s?.Name == name),
-                createOrUpdate: (vn, name, subnet) =>
-                {
-                    subnet.Name = name;
-                    if (vn.Subnets == null)
-                    {
-                        vn.Subnets = new List<Subnet> { subnet };
-                        return;
-                    }
-                    var subnetAndIndex = vn
-                        .Subnets
-                        .Select((sn, i) => new { sn, i })
-                        .FirstOrDefault(sni => sni.sn.Name == name);
-                    if (subnetAndIndex != null)
-                    {
-                        vn.Subnets[subnetAndIndex.i] = subnet;
-                        return;
-                    }
-                    vn.Subnets.Add(subnet);
-                });
+                getList: parentModel => parentModel.Subnets,
```

Comment:
breaking change == assembly major version update

Reply:
this one is not a breaking change but internal restructuring to centralize common code and remove code duplication.

Reply:
I think this one is not a breaking change but internal restructuring.

Reply:
The public factory for Create changed the names of its parameters, so I think this is a breakign change.